src/styles/var.scss

```
@@ -0,0 +1,16 @@
+$weekdays: (
+  'mon': 0,
+  'tue': 1,
+  'wed': 2,
+  'thu': 3,
+  'fri': 4,
+  'sat': 5,
+  'sun': 6,
+);
+$gap: 1px;
+$squareDays: 100px;
+$heightDay: 100px;
+$widthDay: 100px;
```

No need to duplicate variables with same values
```suggestion
$daySize: 100px;
```